docker/xud/xud.conf

```diff
@@ -0,0 +1,24 @@
+lndDir = "~/.lnd"
+rpcPort = 8886
+
```

Comment:
The structure you're using is a bit outdated. For example, `rpcPort` is now under `[rpc]`

Reply:
Yes, and can be dropped in this case because 8886 is the default.